Packs/RunZero/Integrations/RunZero/RunZero.py

```diff
@@ -0,0 +1,525 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import urllib3
+# from typing import Dict, Any
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+MAX_RTT = 1_000_000
+DEFAULT_LIMIT = 50
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any XSOAR logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this  implementation, no special attributes defined
+    """
+
+    def asset_search(self, search_str: str = None):
+        url_suffix = f'/org/assets{search_str}' if search_str else '/org/assets'
+        return self._http_request(method='GET',
+                                  url_suffix=url_suffix,
+                                  headers=self._headers)
+
+    def asset_delete(self, asset_ids: list):
+        url_suffix = f'/org/assets/bulk/delete?asset_ids={asset_ids}'
```

Comment:
when using url args it's better to use the `params` arg for `_http_request`, i.e. to provide a `{arg_key: arg_val}` dict.